docs/source/environment.rst

```diff
@@ -52,3 +52,32 @@ The global environment can also be inspected directly.
         },
         'cpu_brand': 'Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz'
     }
+
+Configuration Files
+-------------------
+
+The computational environment defaults can be overridden by configuration files.
+
+Configuration files must be named ``qcengine.yaml`` and stored either in the directory
+from which you run QCEngine, a folder named ``.qcarchive`` in your home directory,
+or in a folder specified by the ``DQM_CONFIG_PATH`` environmental variable.
+Only one configuration file will be used if multiple are available.
+The ``DQM_CONFIG_PATH`` configuration file takes precedence over the current directory,
```

Comment:
Hmm, `DQM_CONFIG_PATH` is from legacy code. @loriab Any thoughts on this environment variable? Will split this out into an issue.

Reply:
I see we spotted the same thing. Since it's affecting fractal and engine, I think `QCA` is good and generic.

Reply:
Should I keep it as DQM for now since you are splitting off another issue?

Reply:
Yes, please leave it for now and we can tackle it later.